src/status_im2/contexts/wallet/edit_account/view.cljs

```diff
@@ -4,52 +4,56 @@
+             :as create-or-edit-account]
+            [status-im2.contexts.wallet.common.sheets.network-preferences.view
+             :as network-preferences]
             [status-im2.contexts.wallet.edit-account.style :as style]
             [utils.i18n :as i18n]
             [utils.re-frame :as rf]))
 
+(defn- show-save-account-toast
+  [updated-key theme]
+  (let [message (case updated-key
+                  :name  :t/edit-wallet-account-name-updated-message
+                  :color :t/edit-wallet-account-colour-updated-message
+                  :emoji :t/edit-wallet-account-emoji-updated-message
+                  nil)]
+    (rf/dispatch [:toasts/upsert
+                  {:id         :edit-account
+                   :icon       :i/correct
+                   :icon-color (colors/resolve-color :success theme)
+                   :text       (i18n/label message)}])))
+
 (defn- save-account
-  [{:keys [theme type value account]}]
-  (let [edited-account-data (assoc account type value)
-        message             (case type
-                              :name  :t/edit-wallet-account-name-updated-message
-                              :color :t/edit-wallet-account-colour-updated-message
-                              :emoji :t/edit-wallet-account-emoji-updated-message
-                              nil)
-        callback            #(rf/dispatch [:toasts/upsert
-                                           {:id         :edit-account
-                                            :icon       :i/correct
-                                            :icon-color (colors/resolve-color :success theme)
-                                            :text       (i18n/label message)}])]
+  [{:keys [account updated-key new-value theme]}]
+  (let [edited-account-data (assoc account updated-key new-value)]
     (rf/dispatch [:wallet/save-account
-                  {:account  edited-account-data
-                   :callback callback}])))
+                  {:account    edited-account-data
+                   :on-success #(show-save-account-toast updated-key theme)}])))
 
 (defn- view-internal
   [{:keys [theme]}]
   (let [edited-account-name  (reagent/atom nil)
         show-confirm-button? (reagent/atom false)
         on-change-color      (fn [edited-color {:keys [color] :as account}]
                                (when (not= edited-color color)
-                                 (save-account {:account account
-                                                :type    :color
-                                                :value   edited-color
-                                                :theme   theme})))
+                                 (save-account {:account     account
+                                                :updated-key :color
+                                                :new-value   edited-color
+                                                :theme       theme})))
```

Comment:
We received `:type` and when destructuring it was overriding `clojure.core/type`, IMO it's clearer `:updated-key` according to the problem solved in this view.